ui/packages/consul-ui/app/components/consul/server/card/skin.scss

```diff
@@ -22,8 +22,8 @@
   color: rgb(var(--tone-green-800));
 }
 %consul-server-card:not(.voting-status-non-voter) .health-status:not(.healthy) + dd {
-  background-color: rgb(var(--tone-red-050));
-  color: rgb(var(--tone-red-500));
+  background-color: var(--token-color-border-critical);
```

Comment:
the figma specifies the bg color should be `red-010`, not `red-050`, so i'll follow the design and change the value here